app/helpers/profile_helpers.py

```diff
@@ -123,7 +123,7 @@ def _obtain_nb_points_per_segment_no_loss(distances, nb_points_total, total_dist
 
         nb_points_segments[min_index] = (-0.5, nb_points_segments[min_index][1])
         nb_points_segments[max_index] = (-0.5, nb_points_segments[max_index][1] + 1.0)
-        sum_int = sum([int(nbp[0]) for nbp in nb_points_segments])
+        sum_int = sum((int(nbp[0]) for nbp in nb_points_segments))
```

Comment:
dito, and here a `sum_int = sum(int(nbp[0]) for nbp in nb_points_segments)` could probably do, too.